services/skipchain/skipchain.go

```diff
@@ -320,7 +320,7 @@ func (s *Service) startBFTSignature(block *SkipBlock) error {
 		if err := block.BlockSig.Verify(network.Suite, el.Publics()); err != nil {
 			return errors.New("Couldn't verify signature")
 		}
-	case <-time.After(time.Second * 3):
+	case <-time.After(time.Second * 10):
```

Comment:
This kind of stuff is really nasty because we never really know how long. 10 seconds still looks too short for me, as a signing operation could really take 10 secs with latency, extreme end points geographically, etc. Otherwise, if something has not returned after let's say 1mn or 2mn, then it's definitely something wrong.